_opensearch/remote.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Remote-backed storage
+nav_order: 19
+---
+
+# Remote-backed storage
+
+Remote-backed storage offers OpenSearch users a new way to protect against data loss by automatically creating backups of all index transactions and sending them to a supported cloud storage service. In order to expose this feature, segment replication must also be enabled. See [Segment replication]({{site.url}}{{site.baseurl}}/opensearch/segment-replication/) for additional information.
```

Comment:
Few comments on this:

1. Remote store need not be a cloud storage service. We need to re-word `supported cloud storage service`.
2. For more information of the feature itself, should we add links to feature and design proposal?

Reply:
Great call-outs Sachin!  I'll make those changes.

Reply:
For point 2, a link to the feature proposal is already included.  This was how the docs and product teams agreed to label experimental features. It's in the note on line 11.